src/include/kdbplugin.h

```diff
@@ -86,6 +86,9 @@ typedef enum
 /** Everything went fine and the function **did not** update the given keyset/configuration */
 #define ELEKTRA_PLUGIN_STATUS_NO_UPDATE 0
 
+/** Everything went fine and we have a cache hit */
+#define ELEKTRA_PLUGIN_STATUS_CACHE_HIT 2
```

Comment:
Is it necessary to distinguish between NO_UPDATE and CACHE_HIT?

Reply:
Yes. If all backends return:

- NO_UPDATE: nothing at all to do.
- CACHE_HIT: need to fetch from cache (invoke cache, update all pointers)
- SUCCESS: need to fetch from storage

NO_UPDATE is faster than fetching the cache. I need to implement delaying the pointer update though.